libraries/AP_Math/matrix_alg.cpp

```diff
@@ -286,10 +290,14 @@ bool inverse4x4(float m[],float invOut[])
     uint8_t i;
 
 #if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    #if defined(__APPLE__) && defined(__MACH__)
+    _MM_SET_EXCEPTION_MASK(_MM_GET_EXCEPTION_MASK() & ~_MM_MASK_INVALID);
+    #else
     int old = fedisableexcept(FE_OVERFLOW);
     if (old < 0) {
         hal.console->printf("inverse4x4(): warning: error on disabling FE_OVERFLOW floating point exception\n");
     }
+    #endif
```

Comment:
Since we have two different implementations, I think we should have functions for that. That way we can avoid the boilerplate code in future uses.